src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2883,4288 +2899,4153 @@ namespace UnitarySystems {
+        std::string loc_m_HeatingSAFMethod = input_data.heating_supply_air_flow_rate_method;
+        Real64 loc_m_HeatingSAFMethod_SAFlow = input_data.heating_supply_air_flow_rate;
+        Real64 loc_m_HeatingSAFMethod_SAFlowPerFloorArea = input_data.heating_supply_air_flow_rate_per_floor_area;
+        Real64 loc_m_HeatingSAFMethod_FracOfAutosizedHeatingSAFlow = input_data.heating_fraction_of_autosized_heating_supply_air_flow_rate;
+        Real64 loc_m_HeatingSAFMethod_FlowPerHeatingCapacity = input_data.heating_supply_air_flow_rate_per_unit_of_capacity;
+        std::string loc_m_NoCoolHeatSAFMethod = input_data.no_load_supply_air_flow_rate_method;
+        Real64 loc_m_NoCoolHeatSAFMethod_SAFlow = input_data.no_load_supply_air_flow_rate;
+        Real64 loc_m_NoCoolHeatSAFMethod_SAFlowPerFloorArea = input_data.no_load_supply_air_flow_rate_per_floor_area;
+        Real64 loc_m_NoCoolHeatSAFMethod_FracOfAutosizedCoolingSAFlow = input_data.no_load_fraction_of_autosized_cooling_supply_air_flow_rate;
+        Real64 loc_m_NoCoolHeatSAFMethod_FracOfAutosizedHeatingSAFlow = input_data.no_load_fraction_of_autosized_heating_supply_air_flow_rate;
+        Real64 loc_m_NoCoolHeatSAFMethod_FlowPerCoolingCapacity =
+            input_data.no_load_supply_air_flow_rate_per_unit_of_capacity_during_cooling_operation;
+        Real64 loc_m_NoCoolHeatSAFMethod_FlowPerHeatingCapacity =
+            input_data.no_load_supply_air_flow_rate_per_unit_of_capacity_during_heating_operation;
+        Real64 loc_DesignMaxOutletTemp = input_data.maximum_supply_air_temperature;
+        Real64 loc_m_MaxOATSuppHeat = input_data.maximum_outdoor_dry_bulb_temperature_for_supplemental_heater_operation;
+        std::string loc_condenserInletNodeName = input_data.outdoor_dry_bulb_temperature_sensor_node_name;
+        Real64 loc_m_MaxONOFFCyclesperHour = input_data.maximum_cycling_rate;
+        Real64 loc_m_HPTimeConstant = input_data.heat_pump_time_constant;
+        Real64 loc_m_OnCyclePowerFraction = input_data.fraction_of_on_cycle_power_use;
+        Real64 loc_m_FanDelayTime = input_data.heat_pump_fan_delay_time;
+        Real64 loc_m_AncillaryOnPower = input_data.ancillary_on_cycle_electric_power;
+        Real64 loc_m_AncillaryOffPower = input_data.ancillary_off_cycle_electric_power;
+        Real64 loc_m_DesignHRWaterVolumeFlow = input_data.design_heat_recovery_water_flow_rate;
+        Real64 loc_m_MaxHROutletWaterTemp = input_data.maximum_temperature_for_heat_recovery;
+        std::string loc_heatRecoveryInletNodeName = input_data.heat_recovery_water_inlet_node_name;
+        std::string loc_heatRecoveryOutletNodeName = input_data.heat_recovery_water_outlet_node_name;
+        std::string loc_m_DesignSpecMultispeedHPType = input_data.design_specification_multispeed_object_type;
+        std::string loc_m_DesignSpecMultispeedHPName = input_data.design_specification_multispeed_object_name;
```

Comment:
Are you just reading these off for convenience?

Reply:
Yes, this was for speed of implementation.